kratos/geometries/line_2d_2.h

```diff
@@ -792,6 +792,31 @@ class Line2D2 : public Geometry<TPointType>
     }
 
     /**
+     * It computes the normal of the geometry, if possible
+     * @return The normal of the geometry
+     */
+    virtual array_1d<double, 3> Normal() override
```

Comment:
as far as i can see the method Normal is not in the geometry base class API.
this means that it can not be called directly (without an  upcast).

if you have a case for adding it, propose it as a specific issue and we'll discuss it. Indeed in my opinion any change to the base class API should be discussed as an Issue "per se", since it can have profound and unexpected impact on everyone's code.

Note that it may make sense to add a method to compute the normal, simply it should not be done in a generic commit, and only after a discussion